content/posts/jellyfin-10.5.md

```diff
@@ -0,0 +1,109 @@
+
+<video autoplay muted loop preload="none" id="preview">
+    <source src="/images/10.5-release/speed.mp4" type="video/mp4">
+</video>
+
+Not only you're seeing the speed differences, you're seeing the new animations! Images will fade in every time you move to another page. You can disable that in settings if you want to come back the old behaviour though.
+
+The old-standing issue that  many of you had (by the way, introduced by the fellow writer of this post) caused by the images has been fixed in this version,
+although we want to keep improving in that matter, as we're still not completely happy with the current implementation. 
+
+That's pretty much what you'll notice at first sight for web anyway. A lot of not-that-relevant improvements, translations and bug fixes have been added. Consider that they're easter eggs, so happy finding! Ah, and we also added keyboard support for TVs and improved the compatibility with old browsers (just in case)!
+
+Okay, server time then! This time we focused heavily in migrating to .NET Core 3.0 (the framework we use for building the server's core).
+Although this seems boring, let's see some benchmarks from the hex decoder (used in server's tasks):
+
+```
+|                         Method |     Mean |    Error |   StdDev |   Gen 0 | Gen 1 | Gen 2 | Allocated |
+|------------------------------- |---------:|---------:|---------:|--------:|------:|------:|----------:|
+|                New hex decoder | 10.81 us | 0.079 us | 0.074 us |  0.1068 |     - |     - |     336 B |
+|                Old hex decoder | 23.57 us | 0.055 us | 0.051 us | 11.2305 |     - |     - |   35272 B |
+```
+*Shoutout to [Bond-009](https://github.com/Bond-009) for this!*
+
+That's only a tiny example of what we have been cooking and the groundwork preparation we've been doing for upcoming versions.
+However, there are some key improvements here, which are mostly focused in performance:
+
+* Added support for embedded attachments in media files (#1838)
+* Fixed Play To order for playlists
+* Raspbery Pi should have now full hardware-acceleration capabilities when encoding and decoding (older models, not in 4 yet)
+* General improvements to the transcoding speed and reliability
+* Reliability improvements
+* Tons of bug fixes
+
+## Release Notes
+
+All this blog post focused heavily in the server and the web client, as they're the most used ones. However, we also have a bunch of apps that will receive
+this update as well, so [keep an eye](https://github.com/jellyfin) on our GitHub.
+
+Detailed release notes available on the [GitHub's server repository](https://github.com/jellyfin/jellyfin/releases/tag/v10.5.0).
+
+## We're not done yet
+
+As [Joshua explained in his blog post](https://jellyfin.org/posts/jellyfin-in-2019/), we're really excited to see how our whole community is evolving,
```

Comment:
growing, not evolving. Try something like "we are really excited to see our community continue to grow"

Normally informally uses contractions, but "we're really" is offputting because its two "r" sounds back to back, hence the dropping of it.